bool stop_found
float64 dist_to_stop
string sign_nearest
